packages/swingset-liveslots/test/virtual-objects/test-virtualObjectManager.js

```diff
@@ -125,20 +124,29 @@ test('multifaceted virtual objects', t => {
   decr.dec();
   t.is(decr.getCount(), 1);
   const other = makeMultiThing('other');
+
+  flushStateCache();
   t.is(log.shift(), `get kindIDID => undefined`);
   t.is(log.shift(), `set kindIDID 1`);
   t.is(log.shift(), `set vom.vkind.2 {"kindID":"2","tag":"multithing"}`);
   t.is(log.shift(), `set vom.${kid}/1 ${multiThingVal('foo', 1)}`);
+  t.is(log.shift(), `set vom.${kid}/2 ${multiThingVal('other', 0)}`);
   t.deepEqual(log, []);
```

Comment:
This makes for another decent evaluation of the alternative pattern suggested in packages/swingset-liveslots/test/test-cache.js, which would have more useful output in case of failure (since it compares full contents rather than entry-by-entry):
```suggestion
  flushStateCache();
  assertLog(
    `get kindIDID => undefined`,
    `set kindIDID 1`,
    `set vom.vkind.2 {"kindID":"2","tag":"multithing"}`,
    `set vom.${kid}/1 ${multiThingVal('foo', 1)}`,
    `set vom.${kid}/2 ${multiThingVal('other', 0)}`,
  );
```